ttnn/cpp/ttnn/events.hpp

```diff
@@ -0,0 +1,40 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <memory>
+
+#include "tt_metal/impl/device/device_mesh.hpp"
+
+namespace ttnn {
```

Comment:
you can write namespace ttnn::events {
it looks better as for me.

Reply:
done